docs/how-to/rest-apis.txt

```diff
@@ -54,9 +57,9 @@ Sample request:
 
 .. code:: bash
 
-  curl -s "${host}/api/v1/auth/login" \
+  curl -s "${DET_MASTER}/api/v1/auth/login" \
     -H 'Content-Type: application/json' \
-    --data-binary '{"username":"john","password":"2k34h5@$"}' \
+    --data-binary '{"username":"determined","password":""}'
```

Comment:
should we mention that the password is salted and (sha512) hashed? would we provide the password salt here?

Reply:
going w/ an empty password for now :heavy_check_mark: